agenta-web/src/pages/_app.tsx

```diff
@@ -16,6 +17,13 @@ import "ag-grid-community/styles/ag-grid.css"
 import "ag-grid-community/styles/ag-theme-alpine.css"
 import {Inter} from "next/font/google"
 
+const NoMobilePageWrapper = dynamic(
```

Comment:
Let’s modify this to make use of the helper `dynamicComponent` in the`lib/helpers/dynamic.ts` file

Reply:
neat, thanks for the tip @bekossy 🙏